.github/ISSUE_TEMPLATE/feature_request.yaml

```diff
@@ -0,0 +1,35 @@
+name: Feature Request 💡
+description: Have any new idea or new feature ? Please suggest!
+title: "[Feature] <write a small description here>"
```

Comment:
Lets not add `[Feature]` here as we can always assign the labels to the issue.

Can we make a `label` of type `feature` and then automate that label to automatically attach with the feature in this template?